src/commcare_cloud/commcare_cloud.py

```diff
@@ -2,9 +2,14 @@
 from __future__ import print_function
 from __future__ import absolute_import
 
-import inspect
 import os
 
+# set ansible config before other imports so that it's present when `ansible.constants` is imported
+if 'ANSIBLE_CONFIG' not in os.environ:
+    ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+    os.environ['ANSIBLE_CONFIG'] = os.path.join(ROOT_DIR, 'ansible/ansible.cfg')
```

Comment:
I think it's a bit cleaner (but essentially the same) to do it as is done in this line: https://github.com/dimagi/commcare-cloud/blob/72013de466e0ed8328e7b2deed5929c0e6f68cb7/src/commcare_cloud/commands/ansible/helpers.py#L31

I think additionally, if it's done here, then it probably doesn't need to be done there, since it uses the current `os.environ` as its starting point. 

Another pattern we could follow is to put this in another file and then import it like `from commcare_cloud.patch_environ import patch_environ; patch_environ()`. (In loose analogy to e.g. `from gevent import monkey; monkey.patch_all()`.) I think it's a bit nicer, but I don't feel super strongly about it and won't feel bad if you decide not to go with it.

Reply:
Oh, and I realize the line I suggested going off of depends on an import (ANSIBLE_DIR) whereas yours doesn't. I didn't write this down anywhere but I've pretty intentionally made `commcare_cloud.paths` import next to nothing so that it's pretty much always safe to import without circular dependencies or (in this case) having to think about import order and import side effects.

Reply:
Nice, that's definitely cleaner: https://github.com/dimagi/commcare-cloud/pull/2056